tests/nvram_offsets.c

```diff
@@ -21,7 +21,7 @@ int main(void)
     }
 
     /* Check size of auditCommands that expands with new commands */
-#define PD_AUDIT_COMMANDS_EXP_SIZE 14
+#define PD_AUDIT_COMMANDS_EXP_SIZE 15
```

Comment:
OK, will ECC encryption and decryption command be considered to support in the future?

Reply:
We can consider it. Problem by now is that every additional enabled command that alters the format of the state breaks migrateability to older versions of libtpms, which breaks certain scenarios of VM migration. However, due to requests to support runtime-disabled algorithms we *may* have to alter the format of the TPM's state because of that and the ECC encryption stuff could go in with it.